src/sync-accumulator.ts

```diff
@@ -167,22 +168,7 @@ interface IRoom {
     _accountData: { [eventType: string]: IMinimalEvent };
     _unreadNotifications: Partial<UnreadNotificationCounts>;
     _unreadThreadNotifications?: Record<string, Partial<UnreadNotificationCounts>>;
-    _readReceipts: {
-        [userId: string]: {
-            data: IMinimalEvent;
-            type: ReceiptType;
-            eventId: string;
-        };
-    };
-    _threadReadReceipts: {
-        [threadId: string]: {
-            [userId: string]: {
-                data: IMinimalEvent;
-                type: ReceiptType;
-                eventId: string;
-            };
-        };
-    };
+    _receipts: ReceiptAccumulator;
```

Comment:
Is this really something that should be exposed via the interface?
It looks more like implementation specific to me.

Reply:
I didn't want to mess around with the design of the sync accumulator more than I needed to. I haven't really thought about what this interface actually does - let's discuss it sometime.